src/accessors/external_accessor.ts

```diff
@@ -264,6 +273,65 @@ export class ExternalAccessor implements IConsumerApiAccessor {
     await this._httpClient.post<UserTaskResult, any>(url, userTaskResult, requestAuthHeaders);
   }
 
+  // ManualTasks
+  public async getManualTasksForProcessModel(identity: IIdentity, processModelId: string): Promise<ManualTaskList> {
+
+    const requestAuthHeaders: IRequestOptions = this._createRequestAuthHeaders(identity);
+
+    let url: string = restSettings.paths.processModelManualTasks.replace(restSettings.params.processModelId, processModelId);
+    url = this._applyBaseUrl(url);
```

Comment:
```suggestion
    const url = this._applyBaseUrl(urlRestPart);
```

Bitte nicht die Objekte mutieren!

Reply:
es gibt fast nie einen Grund `let` zu nutzen.